novawallet/Modules/NetworkManagement/KnownNetworksList/KnownNetworksListViewController.swift

```diff
@@ -33,6 +33,11 @@ final class KnownNetworksListViewController: UIViewController, ViewHolder {
         super.viewDidLoad()
 
         setup()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
         presenter.setup()
```

Comment:
I would rename ```presenter.setup``` to ```presenter.becameActive``` to explicitly say that it might be called several times during lifecycle